src/storage/ArrayStorage.java

```diff
@@ -0,0 +1,93 @@
+package storage;
+
+import model.Resume;
+
+import java.util.Arrays;
+import java.util.Objects;
+import java.util.logging.Logger;
+
+public class ArrayStorage {
+    private static final Logger log = Logger.getLogger("ArrayStorage Logger");
+
+    private static final int MAX_STORAGE_SIZE = 100000;
+    private final Resume[] storage = new Resume[MAX_STORAGE_SIZE];
+    private int size;
+
+    public void clear() {
+        Arrays.fill(storage, null);
```

Comment:
You don't need to fill all the array with null values

Reply:
Left as is don't know how to resolve. I can recreate the array only